src/develop/masks/masks.c

```diff
@@ -1770,11 +1770,10 @@ void dt_masks_form_remove(struct dt_iop_module_t *module,
         break;
       }
     }
-    if(ok) dt_dev_add_masks_history_item(darktable.develop, module, TRUE);
-    if(ok && module)
+    if(ok)
```

Comment:
Maybe leave a comment here about checking for module == NULL in `dt_dev_add_masks_history_item`?

Reply:
Accepted :-) I stumbled across this while reading the diffs and had to crosscheck.